packages/vector_graphics/lib/vector_graphics.dart

```diff
@@ -339,29 +403,87 @@ class _RenderVectorGraphics extends RenderBox {
     return constraints.smallest;
   }
 
-  final Matrix4 transform = Matrix4.identity();
+  void invalidateRaster() {
+    _lastRasterizedSize = null;
+  }
+
+  // Re-create the raster for a given SVG if the target size
+  // is sufficiently different.
+  Future<void> _maybeUpdateRaster(Size desiredSize) async {
```

Comment:
Wdyt of putting this behind a flag?  It would make it easier to test impact on particular svgs in particular benchmarks.

Reply:
I just switched between branches